integration-cli/docker_cli_by_digest_test.go

```diff
@@ -32,30 +33,23 @@ func setupImageWithTag(c *check.C, tag string) (digest.Digest, error) {
 
 	// tag the image to upload it to the private registry
 	repoAndTag := utils.ImageReference(repoName, tag)
-	if out, _, err := dockerCmdWithError("commit", containerName, repoAndTag); err != nil {
-		return "", fmt.Errorf("image tagging failed: %s, %v", out, err)
-	}
+	out, _, err := dockerCmdWithError("commit", containerName, repoAndTag)
+	c.Assert(err, checker.IsNil, check.Commentf("image tagging failed: %s, %v", out, err))
```

Comment:
The `err` in `check.Commentf` is no need since it will be shown out by the `checker.IsNil` if it was not nil.